bftengine/src/bftengine/ReadOnlyReplica.cpp

```diff
@@ -249,6 +249,9 @@ void ReadOnlyReplica::executeReadOnlyRequest(concordUtils::SpanWrapper &parent_s
   } else {
     LOG_ERROR(GL, "Received error while executing RO request. " << KVLOG(clientId, status));
   }
+
+  ClientReplyMsg replyMsg(0, request.requestSeqNum(), config_.getreplicaId(), status);
+  send(&replyMsg, clientId);
```

Comment:
This needs to be done only in case of error (status != 0)

Reply:
done